### ISSUE TITLE
Jqtree and PHP, the simpler way

### ISSUE THREAD

**Issue by naeco33** [mbraak/jqTree#293]
Hi, 
I'm trying to write a jqtree code with php data, with the simpler way.

I have currently write that :
in the html file :
&lt;html&gt;
&lt;head&gt;
&lt;title&gt;menu simple&lt;/title&gt;
&lt;meta charset="utf-8"&gt;
&lt;script src="jquery.min.js"&gt;&lt;/script&gt;
&lt;script src="tree.jquery.js"&gt;&lt;/script&gt;
&lt;link rel="stylesheet" href="jqtree.css"&gt;
&lt;script type="text/javascript"&gt;

var data = [ { label: 'node1', children: [ { label: 'child1' }, { label: 'child2' } ] }, { label: 'node2', children: [ { label: 'child3' } ] } ];

$(function() {
    $('#tree1').tree({
        //data: data,
        dataUrl: 'menu_content_simple.ajax.php',
        autoOpen: true,
        closedIcon: '&#x25ba;',
        openedIcon: '&#x25bc;',
        onLoadFailed: function(response) {
            console.log("Erreur de load : "+response);
        },
        dragAndDrop: true,
        autoEscape: false,
        keyboardSupport: true,
        openFolderDelay: 500,
        selectable: true,
    });

});
&lt;/script&gt;

&lt;/head&gt;
&lt;body&gt;

&lt;div id="tree1"&gt;&lt;/div&gt;

&lt;/body&gt;
&lt;/html&gt;

in the php file menu_content_simple.ajax.php :
&lt;?php
//header('Content-Type: text/html; charset=utf-8');
header('Content-Type: application/json');

?&gt;
[
    {
        label: 'node1',
        children: [
            {
                label: 'child1'
            },
            {
                label: 'child2'
            }
        ]
    },
    {
        label: 'node2',
        children: [
            {
                label: 'child3'
            }
        ]
    }
]

Firebug say that the JSON data is loaded in 4ms, and it seem correctly formatted as the php file, and it comes in 4ms. I work with a WAMP server. 

I have no result, no tree. And when i uncomment the "data:" and comment the "dataUrl:"  lines, it works.

I have no idee about what i am doing wrong.

I will be great if someone have an idee :)

thank's a lot

PS : i have not found how i can correctly formatted the code in a post here.
